### ISSUE TITLE
Failure to fetch CGPM Resolution data that exists

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-bipm#44]
The data exists right here:
* https://github.com/relaton/relaton-data-bipm/blob/master/data/cgpm/meeting/resolution/1979-06.yaml

But I cannot fetch it:

```sh
$ bundle exec relaton fetch "CGPM -- Resolution 6 (1979)"
[relaton] (CGPM -- Resolution 6 (1979)) not found.
No matching bibliographic entry found
$ bundle exec relaton fetch "BIPM CGPM -- Resolution 6 (1979)"
[relaton] (BIPM CGPM -- Resolution 6 (1979)) not found.
No matching bibliographic entry found
$ bundle exec relaton fetch "BIPM CGPM Resolution 6 (1979)"
[relaton-bipm] ("BIPM CGPM Resolution 6 (1979)") fetching...
[relaton-bipm] ("BIPM CGPM Resolution 6 (1979)") not found.
No matching bibliographic entry found
$ bundle exec relaton fetch "BIPM CGPM-Resolution-1979-06"
[relaton-bipm] ("BIPM CGPM-Resolution-1979-06") fetching...
[relaton-bipm] ("BIPM CGPM-Resolution-1979-06") not found.
No matching bibliographic entry found
```

**Comment by ronaldtse** [comment]
This one as well:
* https://github.com/relaton/relaton-data-bipm/blob/master/data/cipm/meeting/resolution/1879-00.yaml

```sh
$ bundle exec relaton fetch "BIPM CIPM Resolution (1879)"
[relaton] (BIPM CIPM Resolution (1879)) not found.
No matching bibliographic entry found
$ bundle exec relaton fetch "BIPM CIPM -- Resolution (1879)"
[relaton] (BIPM CIPM Resolution (1879)) not found.
No matching bibliographic entry found
$ bundle exec relaton fetch "BIPM CIPM -- Resolution 0 (1879)"
[relaton] (BIPM CIPM Resolution (1879)) not found.
No matching bibliographic entry found
```

**Comment by andrew2net** [comment]
> ```shell
> $ bundle exec relaton fetch "BIPM CIPM Resolution (1879)"
> [relaton] (BIPM CIPM Resolution (1879)) not found.
> No matching bibliographic entry found

This is non correct reference. Should be "BIPM CIPM -- Resolution (1879)".
> $ bundle exec relaton fetch "BIPM CIPM -- Resolution (1879)"
> [relaton] (BIPM CIPM Resolution (1879)) not found.
> No matching bibliographic entry found
> $ bundle exec relaton fetch "BIPM CIPM -- Resolution 0 (1879)"
> [relaton] (BIPM CIPM Resolution (1879)) not found.
> No matching bibliographic entry found
> ```

**Comment by ronaldtse** [comment]
@andrew2net ok after I did `relaton db clear`, this worked:

```sh
bundle exec relaton fetch "BIPM CIPM -- Resolution (1879)"
```

It seems that the db cache mechanism needs some updating. We can't rely on users / metanorma clearing the relaton db to work.

Also, the syntax requiring `--` is not ideal. We ought to support: `CIPM {TYPE} ({YEAR})` as well.

Moreover, this requirement on only using the new format does not make sense. Look at what we have in the bipm-si-brochure:
https://github.com/metanorma/bipm-si-brochure/blob/main/sources/sections-en/99-bibliography.adoc

Are you saying the recent changes to relaton-bipm broke all those references? That's a problem.

**Comment by opoudjis** [comment]
No. I updated to this new reference, when the existing reference in the standoc rspec failed to work.

And it works locally, and it does not work on Github Actions:

https://github.com/metanorma/metanorma-standoc/actions/runs/4268439835/jobs/7430938305

This is broken. I don't know how, but this is broken.

**Comment by opoudjis** [comment]
Urgent, because this is compromising rspec on metanorma-standoc

**Comment by opoudjis** [comment]
Ignore my comments, is now working for me.

**Comment by andrew2net** [comment]
> Also, the syntax requiring `--` is not ideal. We ought to support: `CIPM {TYPE} ({YEAR})` as well.

@ronaldtse we can add this format.

> Are you saying the recent changes to relaton-bipm broke all those references? That's a problem.

Earlier we decided to no support the old format, but if we still need it we can add such references. The relaton-data-bipm has index with multiple references.

**Comment by andrew2net** [comment]
fixed in v 1.14.3 https://github.com/relaton/relaton-bipm#basic-pattern
```
relaton fetch "BIPM CIPM Resolution (1879)"
[relaton-bipm] ("BIPM CIPM Resolution (1879)") fetching...
[relaton-bipm] ("BIPM CIPM Resolution (1879)") found CIPM -- Resolution (1879)
<bibdata type="proceedings" schema-version="v1.2.1">
...
```
